src/tools/compiletest/src/runtest.rs

```diff
@@ -1887,6 +1895,7 @@ impl<'test> TestCx<'test> {
         emit: Emit,
         allow_unused: AllowUnused,
         link_to_aux: LinkToAux,
+        passes: Option<Vec<String>>,
```

Comment:
This could just be a `Vec<String>` and we use `is_empty()` as a signal to dump  all of them.

Reply:
A bunch of tests other than miropt call make_compile_args(), would it be ok to pass Vec::new() to those calls instead of None?